src/marshaller/HipieDDL.js

```diff
@@ -52,17 +52,17 @@
 
     SourceMappings.prototype.doMap = function (item) {
         var retVal = [];
-        for (var key in this.mappings) {
-            try {
+        try {
+            for (var key in this.mappings) {
```

Comment:
i wasn't sure about that one, i didn't know if we wantted the try in the loop or outside, reverting will make the try catch in the loop